web2.0/src/Components/Current/CurrSwell.js

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from 'react';
+import WeatherCard from '../weather-card/WeatherCard';
+
+const CurrSwell = () => {
+  const [currentData, setCurrentData] = useState();
+
+  const getData = async () => {
+    const response = await fetch('http://localhost:8000/swell');
+    const data = await response.json();
+    setCurrentData(data);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+  return (
+    <div>
+      <WeatherCard
```

Comment:
> Consider whether it's possible for you to further refactor your data structure to make it even more generic to support that (e.g., columnName as an additional field, so that you can free yourself from the burden of assigning custom keys to your the WeatherCard that you refactored into a single component (that was a well executed refactor btw, I like it <3).

The refactor benefits would apply here too.